dashboard/modules/serve/serve_agent.py

```diff
@@ -55,7 +55,16 @@ async def get_version(self, req: Request) -> Response:
     @routes.get("/api/serve/deployments/")
     @optional_utils.init_ray_and_catch_exceptions()
     async def get_all_deployments(self, req: Request) -> Response:
-        from ray.serve.schema import ServeApplicationSchema
+        try:
+            from ray.serve.schema import ServeApplicationSchema
+        except ImportError:
```

Comment:
Rather than copy-pasting the same logic across the handlers, can you create a shared decorator?

Reply:
or ideally we could have some kind of shared logic that all routes go through (like a middleware), idk if this HTTP framework can support such a thing